### DISCUSSION TITLE
conflict 관련 질문입니다

### DISCUSSION THREAD

**Discussion by chldppwls12** [chldppwls12/DAYONE_Web#23] [Q&A]
@junhee0110 
지금 __pycache__안의 파일 때문에 confilct가 발생했습니다.
찾아보니 장고를 올릴 때 .pyc, __pycache__는 별도로 안올려도 되는 것 같습니다.
.gitignore 저 파일들을 추가하고 먼저 merge하면 나머지 pull requests들도 충돌이 안날 것 같은데 정확한지 잘 모르겠습니다...
혹시 어떻게 생각하시나요??

**Comment by IamJunhee** [comment]
헐 너무 늦게 확인했네요 ㅠㅠㅠ
.gitignore에 추가하면 나중에 생성되는 파일들은 무시가 되지만 이미 add된 녀석들은 남아있습니다!
그래서 어떤 소소한 작업을 거처야 완전히 무시시킬 수 있습니다!

올리신 PR도 확인했습니다 ㅠㅠ .gitignore 수정 merge하겠습니다.
